### PR TITLE
LPS-153135 Test Autom. StructuredContentAssetPriority#StructuredContentDraftByVersionContainsSetPriority

### PR DESCRIPTION
**Pull request by mateusralv**
Background
Create automation test StructuredContentAssetPriority#StructuredContentDraftByVersionContainsSetPriority

Test Plan
Given a structuredContent draft with Headless Admin Content POST "/v1.0/sites/{siteId}/structured-contents/draft" with a priority field set (as in the json example attached) is created
When invoking /v1.0/structured-contents/{structuredContentId}/by-version/{version}
Then the response body includes the set priority field

JIRA Ticket:
https://issues.liferay.com/browse/LPS-153135

### CODE REVIEW

**Comment by mateusralv** [comment]
ci:test:headless-frontend-functional:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/StructuredContentAssetPriority.testcase [hunk 1]
```diff
@@ -33,6 +33,35 @@ definition {
 		}
 	}
 
+	@priority = "5"
+	test StructuredContentDraftByVersionContainsSetPriority {
+		property portal.acceptance = "true";
+
+		task ("When in Headless Admin Content I create a structuredContent draft with a priority field set") {
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/StructuredContentAssetPriority.testcase:40]: This one is a Given step in the test scenario

**Review by magjed4289 — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro [hunk 2]
```diff
@@ -231,6 +231,23 @@ definition {
 		return "${curl}";
 	}
 
+	macro _versionStructuredContent {
+		Variables.assertDefined(parameterList = "${versionvalue},${responseToParse}");
+
+		var structuredContentId = JSONUtil.getWithJSONPath("${responseToParse}", "$.['id']");
+		var portalURL = JSONCompany.getDefaultPortalURL();
```
[the remark this answers is not in the document]
Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro]: Done!

**Comment by mateusralv** [comment]
ci:test:headless-frontend-functional:nocompile

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**